src/Routes/missions.js

```diff
@@ -0,0 +1,20 @@
+import React, { useEffect } from 'react';
+import { useDispatch } from 'react-redux';
+import MissionChart from '../components/missionChart';
+import { getMissions } from '../redux/mission/missionSlice';
+
+function Missions() {
+  const dispatch = useDispatch();
+
+  useEffect(() => {
+    dispatch(getMissions());
+  }, [dispatch]);
+
+  return (
+    <main>
+      <MissionChart />
+    </main>
+  );
+}
+
+export default Missions;
```

Comment:
- [ ] Your Missions component does not maintain the reservation of missions. From the demo video below, if I reserve missions and navigate out and back to the page, the reservations are removed. I think this may have to do with the issue of fetching the API every time someone visits and dispatching the newly fetched missions to the store.

  [screen-capture (4).webm](https://github.com/harshi0102/space-hub/assets/36542986/13a01762-7a06-430b-bf21-925c6f10b754)